### PR TITLE
Stuff for KyOSS

### PR DESCRIPTION
**Pull request by jsquyres**
Here's a great description of what I want to do.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,8 @@
 Hello!
+
+Hello again!
+
+Goodbye!
+
+
+Yo yo yoy.
```
Comment (zeroasterisk) [inline README.md]: this makes no sense...

File: README.md [hunk 2]
```diff
@@ -1 +1,8 @@
 Hello!
+
+Hello again!
```
Comment (zeroasterisk) [inline README.md]: hey back

File: README.md (hunk 1 above)
Comment (jsquyres) [inline README.md]: Yes it does!  You suck!

Comment (zeroasterisk) [inline README.md]: ok ok - you win.

**Pull request merged**